src/Logger.php

```diff
@@ -271,9 +262,9 @@ protected function parseMessage(string $message, array $context): string
      * request ($_SERVER['REQUEST_TIME_FLOAT']).
      * @return float the total elapsed time in seconds for current request.
      */
-    public function getElapsedTime(): float
+    public static function getElapsedTime(): float
     {
-        return \microtime(true) - $_SERVER['REQUEST_TIME_FLOAT'];
+        return \microtime(true) - ($_SERVER['REQUEST_TIME_FLOAT'] ?? 0);
```

Comment:
That will give totally unpredicatable result in case `$_SERVER['REQUEST_TIME_FLOAT']` is not available.